python/GafferSceneUI/SceneInspector.py

```diff
@@ -482,7 +482,7 @@ def _formatValues( self, values ) :
 			return self.__formatNumbers( values )
 		elif isinstance( values[0], basestring ) :
 			return self.__formatStrings( [ str( v ) for v in values ] )
-		elif isinstance( values[0], IECoreScene.PrimitiveVariable ) :
+		elif isinstance( values[0], IECore.CompoundData ) :
```

Comment:
Let's do as we did for the creases, and use a custom TextDiff subclass instead of have TextDiff know about primitive variables. That way we're one step closer to being able to move TextDiff to GafferUI.